tests/data/wzdx.json

```diff
@@ -1,11 +1,11 @@
 {
-  "road_event_feed_info": {
+  "feed_info": {
     "feed_info_id": "49253be7-0c6a-4a65-8113-450f9041f989",
     "update_date": "2022-02-14T21:41:33Z",
     "publisher": "CDOT",
     "contact_name": "Ashley Nylen",
```

Comment:
Since this is in our tests/data directory, are we testing against these values? We still have Ashley in here, does that need updated?

Reply:
There are a significant number of sample files and test files. I updated a few of the more obvious ones, but I am creating a backlog task to go through and regenerate them from data available today, with new dates and everything: https://dev.azure.com/SOC-OIT/CDOT/_backlogs/backlog/CDOT%20Connected%20Vehicles/Backlog%20items/?workitem=218691